src/super_gradients/training/datasets/samplers/infinite_sampler.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*-
+# Copyright (c) Megvii, Inc. and its affiliates.
```

Comment:
what? what license is this?

Reply:
added license in https://github.com/Deci-AI/super-gradients/commit/207848406e975b66dfeb5eaa801a0047ebefb670